website/content/en/v1.2/concepts/disruption.md

```diff
@@ -358,6 +358,22 @@ spec:
         karpenter.sh/do-not-disrupt: "true"
 ```
 
+Pods with PDBs will not be gracefully evicted by the [Termination Controller]({{<ref "#termination-controller">}}) or be considerd for consolidation. When multiple pods on a node have different PDBs, all PDB constraints must be satisfied simultaneously for Karpenter to consolidate the node. This can create complex eviction scenarios:
```

Comment:
Planning on back porting these change to all version of the docs. I want to agree on the wording first